vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml

```diff
@@ -21,7 +21,10 @@
                 <constructor-arg>
                     <bean parent="httpClientConfig">
                         <property name="circularRedirectsAllowed" value="${http.redirects-provider.circular-redirects-allowed}" />
-                        <property name="credentials" value="${http.redirects-provider.credentials}" />
+                        <property name="username"
+                            value="#{'${http.redirects-provider.credentials}'.length() == 0 ? null : T(org.apache.commons.lang3.StringUtils).split('${http.redirects-provider.credentials}', ':', 2)[0]}" />
+                        <property name="password"
+                            value="#{'${http.redirects-provider.credentials}'.length() == 0 ? null : T(org.apache.commons.lang3.StringUtils).split('${http.redirects-provider.credentials}', ':', 2)[1]}" />
```

Comment:
please file a ticket to remove this property in `0.4.0`

Reply:
done https://github.com/vividus-framework/vividus/issues/1813